### ISSUE TITLE
Travis-CI failure: python 3.5 segmentation fault

### ISSUE THREAD

**Issue by mroberge** [mroberge/hydrofunctions#27]
* HydroFunctions version: 0.1.6 all branches
* Python version: 3.5
* Operating System: Linux (Travis-ci build)
* Travis Job [#184](https://travis-ci.org/mroberge/hydrofunctions/jobs/379854009)

### Description

Travis-ci suddenly exits during the first test with a segmentation fault. This does not affect Python 3.4 or 3.6.

### What I Did

```
python setup.py test

...
running build_ext
test_charts_cycleplot_exists (tests.test_charts.TestCyclePlot) ... /home/travis/.travis/job_stages: line 57:  1794 Segmentation fault      (core dumped) python setup.py test
```

**Comment by mroberge** [comment]
I required `typing==3.6.4` and this didn't affect the errors in my [3.5 build #186](https://travis-ci.org/mroberge/hydrofunctions/jobs/379892777) from the `segmentation fault` described above in [184.2](https://travis-ci.org/mroberge/hydrofunctions/jobs/379854009). See commit e587fb5a6f58b94207ef1d9a334cb07e46bcce0a

**Comment by mroberge** [comment]
I updated the Develop branch with the following fixes in commit bd30b73d9fef021c59eacf7aee772650575bcf2d:

- don't upgrade from Pandas 0.22.0 to 0.23.0
       - runs with Pandas 0.23.0 included ended with RecursionErrors
       - using the old Pandas fixes the Python 3.6 build
       - Python 3.4 never used Pandas 0.23.0, and it was always able to build.
       - Python 3.5 still doesn't pass: It now reaches the tests though, but gets a segmentation fault.
- Don't test for Python 3.5 
      - `python setup.py test` errors out due to a Segmentation Fault (described above)
      - pytest errors during tests because it can't import matplotlib.

**You can see the results in [TravisCI build #190](https://travis-ci.org/mroberge/hydrofunctions/builds/380280719)**

I'm still working on this issue in the `require-old-pandas22` branch.
The `remove-draw-map` branch wasn't successful at fixing anything. It can be deleted.

**Comment by mroberge** [comment]
I finally resolved this issue by not including Seaborn and Bokeh in my build. I have no idea why this worked; the errors never mention seaborn.
